Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Loading.md

![Loading](./media/loading.png)

> ## Add namespace using
> Loading is defined in namespace Tizen.NUI.Components.

Reviewer: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement loading, add namespace `using Tizen.NUI.Components` in your application:
```
